Sources/ChatGPTAPIManager/ChatGPTAPIManager.swift

```diff
@@ -206,6 +208,57 @@ final public class ChatGPTAPIManager {
         }
     }
     
+    /// Requests audio transcription based on the provided parameters.
+    ///
+    /// - Parameters:
+    ///   - fileUrl: The URL of the audio file to be transcribed.
+    ///   - prompt: The prompt or context for the transcription.
+    ///   - temperature: (Optional) The temperature value for generating diverse transcriptions. Defaults to nil.
+    ///   - language: (Optional) The language of the transcription. Defaults to nil.
+    ///   - model: (Optional) The model to be used for transcription. Defaults to ChatGPTModels.
+    ///   - endPoint: The URL endpoint for the transcription service.
+    ///   - completion: A completion handler to be called with the result of the transcription request.
+    ///                 The handler takes a Result object, which contains either the transcribed text or an error.
+    ///                 Use the `.success` case to access the transcribed text and the `.failure` case to handle errors.
+    public func audioTranscriptionRequest(fileUrl: URL, prompt: String, temperature: Double? = nil, language: String? = nil, model: ChatGPTModels, endPoint: APPURL, completion: @escaping (Result<String, Error>) -> Void) {
```

Comment:
Make model param optional by assigning the default value.
Set others params default value as well which is not required.